index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700;900&display=swap"
+        rel="stylesheet">
+    <title>Nana's portfolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="header-wrapper">
+            <header>
```

Comment:
Please follow the design for the Toolbar section.
![nav](https://user-images.githubusercontent.com/35504149/138158644-00f7077a-c9a8-4926-9313-f45ca289c009.png)